opennlp-tools/src/test/java/opennlp/tools/formats/ad/ADSentenceSampleStreamTest.java

```diff
@@ -54,16 +54,17 @@ public void setup() throws IOException {
     InputStreamFactory in = new ResourceAsStreamFactory(ADSentenceSampleStreamTest.class,
         "/opennlp/tools/formats/ad.sample");
 
-    ADSentenceSampleStream stream = new ADSentenceSampleStream(
-        new PlainTextByLineStream(in, "UTF-8"), true);
+    try (ADSentenceSampleStream stream = new ADSentenceSampleStream(
+          new PlainTextByLineStream(in, "UTF-8"), true)) {
 
-    SentenceSample sample = stream.read();
+      SentenceSample sample = stream.read();
 
-    while (sample != null) {
-      System.out.println(sample.getDocument());
-      System.out.println("<fim>");
-      samples.add(sample);
-      sample = stream.read();
+      while (sample != null) {
```

Comment:
That is bad style to read from a stream, it can assign directly in the while condition